apps/site/src/lib/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import hack from "@/assets/logos/hack.svg";
+import mail from "@/assets/logos/mail.svg";
+import facebook from "@/assets/logos/facebook.svg";
+import instagram from "@/assets/logos/instagram.svg";
+import linkedin from "@/assets/logos/linkedin.svg";
+import youtube from "@/assets/logos/youtube.svg";
+import Image from "next/image";
+
+import styles from "./Footer.module.scss";
+
+const Footer = () => {
+
+	const imageArray = [
+	[hack, "Hack Logo"],
+	[mail, "Email"],
+	[facebook, "Facebook"],
+	[instagram, "Instagram"],
+	[linkedin, "Linkedin"],
+	[youtube, "Youtube"]]
+
+	return (
+	<footer className={`${styles.footer} flex flex-col items-center`}>
+		<div className='flex flex-row footer-logos items-center gap-10 mt-12'>
+			{imageArray.map((item, index) => {
+				return(
+					<a>
```

Comment:
Sorry, I realized that I didn't even provide you with the links to our social media icons. Here are the links:
- Hack icon: https://hack.ics.uci.edu
- Mail icon: mailto:hack@uci.edu
- Facebook: https://www.facebook.com/groups/HackAtUCI/
- Instagram: https://www.instagram.com/hackatuci/
- LinkedIn & YouTube: we actually decided not to include these, so please remove them.

Reply:
As such, please update the `<a>` tags accordingly by providing the links as values to the `href` attribute.

Reply:
Additionally, the deployment fails because of each element should have a `key` prop. You can read more about it [here](https://react.dev/learn/rendering-lists).